src/views/bav/could-not-match.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base-form.njk" %}
+{% set hmpoPageKey = "couldNotMatch" %}
+{% set hmpoTitle = couldNotMatch.title %}
+{% set gtmJourney = "bav - cannotProceed" %}
+{% set backLink = false %}
+
+{% from "hmpo-text/macro.njk" import hmpoText %}
+{% from "hmpo-radios/macro.njk" import hmpoRadios %}
+{% from "hmpo-form/macro.njk" import hmpoForm %}
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+
+{% block mainContent %}
+
+{% set title = translate("couldNotMatch.title") %}
+{% set content = translate("couldNotMatch.content") %}
+{% set prompt = translate("couldNotMatch.prompt") %} 
+{% set formInstructions = "<p>"+content+"</p><p class=\"couldNotMatchPrompt\">"+prompt+"</p>"%}
```

Comment:
I've never seen the the + symbols before, but I guess they are a nunjucks thing? What do they do?

Reply:
Yes this is a nunjucks thing, it's a way of using paragraphs with different font sizes that can then be used as one 'formInstructions' concatenated string in the hmpoForm call below which ensures the text works with accessibility screen readers

Reply:
Oh as in it's the only way to add our custom class, or?

Reply:
That complies with DAC accessibility yes I believe so

Reply:
What part of DAC accessibility? I'm just trying to understand what exactly we are trying to achieve here in case I need to do it in the future

Reply:
I think that by having the three strings concatenated and set to the formInstructions variable, when formInstructions is used as the value for the hint property in the couldNotMatchChoice radio fieldset it allows for multiple lines and font sizes to be used. The hint property is the only part where we can write copy that is associated with the fieldset so it needs to be assembled from its constituent parts before being used in the fieldset. It's actually not accessibility related, that is the title not the hint sorry - this is a formatting solution